### ISSUE TITLE
Images with text should be accessible to screen readers

### ISSUE THREAD

**Issue by gurmukhp** [fejes713/accessibility-guide#19]
Great article. Thanks for creating.

I was quite interested to read the following under this heading: 

> The most straightforward way to do this is to style a header element as image and use text-indent: -9999px as its text content.

The [second rule of 'using aria'](https://www.w3.org/TR/using-aria/#secondrule) says "Do not change native semantics, unless you really have to." which seems to contradict your recommendation. 

Would you mind clarifying what you meant, a code example would be great (esp on the site)

Thanks

**Comment by fejes713** [comment]
Hey Gurmukh,

I am glad that you like the project 🤗 

The recommendations on the https://webaccessibility.guide aren't tied to each other. Some developers tend to use `<h1>` tags on heading for their company logo and this rule is focused towards them. This is an outdated approach but it is still being used.

The old **bad** way of doing it is the `<h1>` is being styled as an image with no text. The CSS for that would at minumum be the following:

`background: transparent url("logo.png") no-repeat scroll 0% 0%;`

The workaround with this approach is to include the text in `h1` but to add `text-indent: -9999px` to it. That way screen reader would read the actual text, but the normally sighted user won't be able to see it (only the logo will be present). The code would look similar to the one at https://www.sonos.com/en/home - However Sonos, in this case, included SVG inside `h1` and provided users with `aria-label` so they're 100% accessible.

The **good** way of doing this is, as you've said it, to use correct semantics and use `img` instead of `h1` tag. Images with the proper `alt` tag are accessible so that is best solution definitely.

Thanks for pointhing this out! Since @csswizardry mentioned this card as well do you think it would be a good idea to remove it as well? It makes more confusion than help.

From the UX side 😉 do you think design and usability are fine?

**Comment by gurmukhp** [comment]
Hey!

Yeah I think it'd be good to remove that card to avoid confusion. If you could add code samples that'd help too.

The site is great, I like how you've bundled all JS and CSS into one file each and it loads super quick. Nice work!

**Comment by fejes713** [comment]
Done 😉 

Thanks for the feedback @gurmukhp if you happen to have any new ideas I am looking forward to hearing from you!
